core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -1307,13 +1316,13 @@ private String redirectOrOriginal(String contentUrl) {
   private void openRandomArticle() {
     String articleUrl = zimReaderContainer.getRandomArticleUrl();
     Log.d(TAG_KIWIX, "openRandomArticle: " + articleUrl);
-    openArticle(articleUrl);
+    openArticle(articleUrl, false);
   }
 
   @OnClick(R2.id.bottom_toolbar_home)
   public void openMainPage() {
     String articleUrl = zimReaderContainer.getMainPage();
-    openArticle(articleUrl);
+    openArticle(articleUrl, false);
```

Comment:
A way around extending `openArticle` without having to change all the call sites would have been to have 2 versions of the method, a 1 arg and a 2 arg
```
 private void openArticle(String articleUrl, boolean isInNewTab) {
//all the stuff it does
}

private void openArticle(String articleUrl){
  openArticle(articleUrl, false)
}
```
this would be musch simpler in kotlin where you can have default parameters but we are unfortunately a while away from converting this to kotlin